packages/eslint-plugin/src/config/by-project-type/monorepo-root.ts

```diff
@@ -1,6 +1,7 @@
-import type { Linter } from "eslint";
 import { mdxFiles, sourceFiles } from "../../utils/patterns";
```

Comment:
Are you imports automatically sorted like that or is that a personal preference you have and you manually do in files?

Reply:
Automatically sorted with VSCode

Reply:
my editor doesnt format vscode import like that. Can you share your config or even add it to our vscode settings files?

Reply:
Resolved in DM